.jscsrc

```diff
@@ -60,7 +60,7 @@
   "disallowSpacesInsideParentheses": true,
 
 
-  "validateJSDoc": {
+  "jsDoc": {
```

Comment:
validateJSDoc is now deprecated: https://github.com/jscs-dev/node-jscs/issues/993
See: http://jscs.info/rule/jsDoc.html